ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js

```diff
@@ -1086,6 +1086,10 @@ if (typeof window.DatePickerWidget === 'undefined') {
         if (clearText) {
             defaultOptions.locale.clearText = clearText;
         }
+        var zero = FormView.LanguageUtils.getTranslatedString(locale, "0");
```

Comment:
Please add a test case for this

Reply:
@devgurjar To confirm - this was documented - but not implemented? https://github.com/adobe/aem-core-forms-components/pull/990/files#diff-762b951cd2955c5f45991f718c78ceeae34eb41c962569de26f1c6f753361ccfR44

Reply:
Can we add a test case for this part. Also - this is independent of current Jira - or customer requires this as part of the change as well?

Reply:
This is independent of current jira, customer has not asked for this part

Reply:
I have added test cases for Zero as well